examples/research_projects/jax-projects/README.md

```diff
@@ -116,7 +118,232 @@ Additionally, the organizers, other participants, or anybody in the community re
+summarization model exists already for CNN/Daily mail and pretty much no code has to be 
+written. 
+A well-defined project does not need to have the dataset be part of 
+the `datasets` library and the training script already be pre-written, however it should 
+be clear how the desired dataset can be accessed and how the training script can be 
+written. 
+
+It is also important to have a clear plan regarding the work flow. Usually, the 
+data processing is done in a first step. Once the data is in a format that the model can 
+work with, the training script can be written, etc. These steps should be more detailed 
+once the team has a clearly defined project. It can be helpful to set deadlines for each step.
+
+### Workload division
+
+To effectively work as a team, it is crucial to divide the workload among everybody.
+Some team members will be more motivated and experienced than others and 
+some team members simply want to participate to learn more and cannot contribute that 
+much to the team. This is totally fine! One cannot expect everybody in the team to have the same level of experience and time/motivation during the community week.
+
+As a conclusion, being honest about one's expected involvement is crucial so that 
+the workload can be divided accordingly. If someone doesn't think her/his tasks are feasible - let 
+the team know early on so that someone else can take care of it!
+
+It is recommended that the motivated and experienced team members take the lead in dividing the work and are ready to take over the work of any another team member. 
+
+The workload can often be divided according to:
+
+- data preprocessing (load the data and preprocess data in the correct format)
+- data tokenization / data collator (process data samples into tokens or images)
+- model configuration (writing the code that defines the model)
+- model forward pass (make sure input / output work correctly)
+- loss function (define the loss function)
+- putting the pieces together in a training script
+
+Many of the steps above require other steps to be finished, so it often makes sense 
+to use dummy data in the expected format to start, *e.g.*, with the model forward pass 
+before the data preprocessing is done.
+
+### Expectations
+
+It is also very important to stay realistic with the scope of your project. Each team 
+has access to a TPUv3-8 for only *ca.* 10 days, so it's important to keep the scope of 
+the project reasonable. While we do want each team to work on interesting projects, each 
+team should make sure that the project goals can be achieved within the provided compute 
+time on TPU. *E.g.*, pretraining a 11 billion parameters T5 model is not really a realistic 
+task with just 10 days of TPUv3-8 compute. 
+Also, it might be difficult to finish a project where the whole modeling, dataset and training code has to be written from scratch.
+
+Before, officially defining your project we will try to help your team to make sure that 
```

Comment:
Just a suggestion, we could say that they should feel free to reach out if they need another pair of eyes on the project after they have defined it.